### PR TITLE
Common/pagination media query

### PR DESCRIPTION
**Pull request by PawelOwiesek**

### CODE REVIEW

File: src/common/Pagination/styled.js [hunk 1]
```diff
@@ -7,12 +7,24 @@ export const Container = styled.div`
   align-items: center;
   justify-content: center;
   margin: 40px 0 100px;
```
Comment (MirekSobota) [inline src/common/Pagination/styled.js:9]: I think it need to be `32px ` not `40px`

File: src/common/Pagination/styled.js [hunk 2]
```diff
@@ -7,12 +7,24 @@ export const Container = styled.div`
...
+    gap: 8px;
+  }
 `;
 
 export const Wrapper = styled.div`
   display: flex;
   gap: 12px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    gap: 4px;
```
Comment (MirekSobota) [inline src/common/Pagination/styled.js:24]: I think gap need to be` 8px`

File: src/common/Pagination/styled.js (hunk 1 above)
Comment (PawelOwiesek) [reply src/common/Pagination/styled.js:9]: Why do you think so? In figma, the space between the list of movies and the pagination is 40px.

**Review by MirekSobota — APPROVED** [review]
Looks interesting, and I hope that it will work well with other elements. 🤞🏻

File: src/common/Pagination/styled.js (hunk 2 above)
Comment (PawelOwiesek) [reply src/common/Pagination/styled.js:24]: You're right, I'm correcting.

File: src/common/Pagination/styled.js (hunk 1 above)
Comment (MirekSobota) [reply src/common/Pagination/styled.js:9]: Please take a look on this picture: 

image

Comment (PawelOwiesek) [reply src/common/Pagination/styled.js:9]: This is correct, but for the mobile version. We misunderstood each other a bit😂

**Pull request merged**